### PR TITLE
2019/tasks/robot eye

### PR DESCRIPTION
**Pull request by justinhart**
Added the Robot Eye task.

### CODE REVIEW

**Comment by kyordhel** [comment]
@justinhart Please edit the content of the PR explaining what this task is about (i.e. TL,DR; version).

File: tasks/RobotEye.tex [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Location:} Living room
+
+	\item \textbf{Clothing:} There is a rack of clothing from which the operator must choose, with shoes in a bin on the floor.
+	\begin{itemize}[nosep]
+		\item\textit{Distractor objects}: There should be distractor objects in this set which are irrelevant to the scenarios named.
+		\item\textit{Target objects}: There should be target objects (more than one per scenario) which fit the scenarios described.
```
Comment (kyordhel) [inline tasks/RobotEye.tex]: The word scenario is reserved in the rulebook for the Party Host and Housekeeper Scenarios.

File: tasks/RobotEye.tex [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+\subsection{Referee instructions}
+The referee needs to
+\begin{itemize}
+	\item Either allow the robot to view the objects and indicate them \textbf{OR}
+	\item Show the objects to the robot for selection (team's choice).
+\end{itemize}
+
+\subsection{OC instructions}
+5 hours before the test:
```
Comment (kyordhel) [inline tasks/RobotEye.tex]: Setup days | 1 day before the test | 2 hours before the test

**Comment by justinhart** [comment]
This probably needs a bit more work before it's ready for RoboCup@Home. I think that the core idea is good, but we probably need to workshop it more and may re-propose it in 2020.

**Comment by kyordhel** [comment]
Based on #536 comments by @swachsmu and @iocchi, I think we can run some analysis on the task you're proposing.

I'm working on a *scientific* rulebook to re-visit all 2019 tests from a scientific perspective (challenges, state of the art research, and contributions), include the roadmaps, and future tests. It's a hell of a work, but I think you can help by writing this and other tests/tasks as research milestones there.

Will let you know when is pushed.

**Comment by MatthijsBurgh** [comment]
Most have been said. I just wanted to add the following.

Computervision, AI and Robotics are pretty close. Challenges should always be focused around Robotics. This challenge feels way too much Computervision and AI.

**Pull request closed without merging**